src/libs/ajax/workflows-app/CbasPact.test.js

```diff
@@ -42,7 +42,6 @@ const TIMESTAMP_REGEX = '[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}\\
 const cbasPact = new PactV3({
   consumer: 'terra-ui',
   provider: 'cbas',
-  log: path.resolve(process.cwd(), 'logs', 'pact.log'),
```

Comment:
I had this argument in my test file and typescript complained, it isnt actually present in PactV3Options so I am removing it here as well. This file is still .js so thats why it isnt complaining.